Source/UI/Coordinators/PasswordCoordinator.swift

```diff
@@ -194,4 +229,72 @@ extension PasswordCoordinator {
         }
         self.navigationController.pushViewController(viewController, animated: true)
     }
+    private func getPasswordFromKeychain(for identifier: Identifier) -> String? {
+        if #available(iOS 11.3, *) {
+            var query = [String: Any]()
+    
+            query[kSecClass as String] = kSecClassGenericPassword
+            query[kSecReturnData as String] = kCFBooleanTrue
+            query[kSecAttrAccount as String] = identifier.normalizedString as CFString
+            query[kSecAttrLabel as String] = "com.schibsted.account.biometrics.secrets" as CFString
+            query[kSecUseOperationPrompt as String] = "Please put your fingers on that button" as CFString
+    
+            var queryResult: AnyObject?
+            let status = withUnsafeMutablePointer(to: &queryResult) {
+                SecItemCopyMatching(query as CFDictionary, UnsafeMutablePointer($0))
+            }
+    
+            if status == noErr {
+                let password = String(data: queryResult as! Data, encoding: .utf8)!
+                print("Password: \(password)")
+                return password
+            } else {
+                print("Authorization not passed")
+                return nil
+            }
+        } else {
+            // Fallback to password login.
+            return nil
+        }
+    }
+    private func updatekeyChain(for identifier: Identifier, loginFlowVariant: LoginMethod.FlowVariant,  password: String, completion: @escaping () -> Void ) {
+        if #available(iOS 11.3, *) {
+            let accessControl = SecAccessControlCreateWithFlags(kCFAllocatorDefault, kSecAttrAccessibleWhenPasscodeSetThisDeviceOnly, .userPresence, nil)
+            var dictionary = [String : Any]()
+            dictionary[kSecClass as String] = kSecClassGenericPassword
+            dictionary[kSecAttrLabel as String] = "com.schibsted.account.biometrics.secrets" as CFString
+            dictionary[kSecAttrAccount as String] = identifier.normalizedString as CFString
+            dictionary[kSecValueData as String] = password.data(using: .utf8)! as CFData
+            dictionary[kSecAttrAccessControl as String] = accessControl
+            
+            let identityNotFirstLogin = UserDefaults.standard.bool(forKey: "identity.isNotFirst.login")
```

Comment:
So the standard `UserDefaults` is shared between the whole app. So you just have to be aware of potential key clashes. There's a `Settings` object just for this purpose in the SDK, which also makes sure that every setting you put in to whatever "storage" the sdk is using (UserDefaults by default), the key is prepended with something like "com.schibsted.identity" and when you retrieve keys this is taken in to account. See the Core/Settings.swift file for details. Also, the `Settings` object is configurable and injectable by the tests as a bonus.

So this would be
```
if let value = Settings.value(forKey: "isFirstLogin") as? Bool, value  {
  // we have a value and it is true
} else {
  // We don't have a value OR it is false
}
```